nuxt/components/form/Form.vue

```diff
@@ -12,13 +12,22 @@
       <div class="flex flex-col min-h-0 overflow-y-auto gap-6">
         <slot />
         <div class="flex flex-col items-center justify-between">
+          <div class="flex justify-center">
+            <NuxtTurnstile v-model="state.turnstileKey" />
```

Comment:
That would be something that can be implemented

Reply:
What do you think is the best way to recognize if a form is checked by turnstile? I would think that a boolean variable can be passed to the form. It can then be either true for an enabled turnstile check or false for a disabled turnstile check

Reply:
Yes! I'd say `isCaptchaEnabled` should be turned off by default and on for the forms that currently use it.

Reply:
Well actually, what now comes to my mind: I think the performance gain is only possible if a separate component is created (a `<FormCaptcha>`). Otherwise the vite wouldn't know if `<NuxtTurnstile v-if="isCaptchaEnabled">` turns true sometime later.

Reply:
So should it be left as it is now?

Reply:
I can take over this optimization

Reply:
Please review my changes (89610348e3ce7c946f3849f422774d20cafcbaa2) and resolve if happy!

Reply:
Are you sure you linked the correct commit hehe

> Please review my changes ([8961034](https://github.com/maevsi/maevsi/commit/89610348e3ce7c946f3849f422774d20cafcbaa2)) and resolve if happy!

Did you link the correct commit here? It seems to me that you meant https://github.com/maevsi/maevsi/pull/1122/commits/b15ed8cd5de5dd5b25a919b024e9bc4b17b12e73

Regarding that commit, thanks for that great submission. I only found one thing that might be a problem. The variable `isVisible` is only false when the nuxt config contains a test site key that makes the widget invisible. In the case when a real site key which is set to invisible is used, the `FormInputCaptcha.vue` component thinks that the widget is visible but in reality it isn't. In that case the elements surrounding the widget would be shown while the widget itself would be invisible. A solution for this should be found.

Reply:
Next try: fc164e2f9d36bb1a07d9b3744da5f59584438812
Please check and resolve this thread if happy :)